app/views/users/show.html.erb

```diff
@@ -1,9 +1,17 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Users Details</title>
-  </head>
-  <body>
-    <h1>Details for a given user</h1>
-  </body>
-</html>
+<div class="flex flex-col w-full">
+  <div id="userInfos">
+        <%= render partial: 'shared/user', locals: {user: @user} %>
+
+        <div id="bio" class="border-4 border-black mt-4 md:w-1/2 mx-auto mb-5">
+          <h2>Bio</h2>
+          <p><%= @user.bio %></p>
+        </div>
+  </div>
+
+  <% @user.posts.each do |post| %>
+    <%= render partial: 'shared/post', locals: {post: post} %>
+  <% end %>
+
+  <button class="mx-auto font-bold w-5/12 md:w-1/10 mb-5 border-solid border-4 border-gray-500">See all posts</button>
```

Comment:
- [ ] This button does not work. When clicked it should take the user to the post index page.